index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="style.css">
+  <script src="./script.js" async defer></script>
+  <meta charset="UTF-8">
+  <title>Tic-Tac-Toe</title>
+</head>
+<body>
+  <h1>Tic-Tac-Toe</h1>
+  <h2 id="turntxt">It is player turn</h2>
+  <button type="button" id="reset">Reset Game</button>
+  <div class="grid-container">    
+    <div class="grid-square" id='sq0'>
```

Comment:
This is a neat way to make your board work. I'd throw an extra challenge at you, to make this board programmatically using js! It would result in nice DRY code